doc/interop-test-descriptions.md

```diff
@@ -681,6 +695,14 @@ When the client requests COMPRESSABLE payload, the response includes a payload
 of the size requested containing all zeros and the payload type is
 COMPRESSABLE.
 
+### Echo Status
+[Echo Status]: #echo-status
+When the client sends a response_status in the request payload, the server returns
```

Comment:
response_status is now in messages.proto. Clients setting response_status in StreamingInputCallRequest should set the same value in all messages in the stream. And the server should be free to use this value from any message in the client->server stream. Using this value from the last received message seems a reasonable server impl.